distributed_shampoo/distributed_shampoo.py

```diff
@@ -394,7 +410,8 @@ def __init__(
         if preconditioner_dtype is not None:
             if precision_config is None:
                 precision_config = PrecisionConfig(
-                    factor_matrix_dtype=preconditioner_dtype
+                    factor_matrix_dtype=preconditioner_dtype,
+                    computation_dtype=preconditioner_dtype,
```

Comment:
I only added this because I removed [this code](https://github.com/facebookresearch/optimizers/pull/27/files#diff-1e1a1ed5cd1ee1bf41faa070a43405e5802c862ad9285abb4b6a80665b8d9035L520-L524), but since `preconditioner_dtype` is depreciated, this code will not be needed anyway.